slides/Combinatorics/Catalan.svg

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Do we actually need this, now that you're using LaTeX to build those formulas? The same can be said for any of the images (except for the Buff pics of course).